server/member/watch_leader.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package member
+
+import (
+	"context"
+	"time"
+
+	"github.com/CeresDB/ceresmeta/pkg/log"
+	"github.com/CeresDB/ceresmeta/server/etcdutil"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.uber.org/zap"
+)
+
+const (
+	WatchLeaderFailInterval = time.Duration(200) * time.Millisecond
+
+	waitReasonFailEtcd    = "fail to access etcd"
```

Comment:
use const iota

Reply:
The reason will be used to log but iota is not readable.